scalability_and_performance/ztp-configuring-single-node-cluster-deployment-during-installation.adoc

```diff
@@ -0,0 +1,54 @@
+:_content-type: ASSEMBLY
+[id="sno-du-deploying-distributed-units-manually-on-single-node-openshift"]
+= Deploying distributed units manually on single node OpenShift
+include::modules/common-attributes.adoc[]
+:context: sno-du-deploying-distributed-units-manually-on-single-node-openshift
+
+toc::[]
+
+
+This procedure tells you how to manually deploy clusters on a small number of single nodes as a distributed unit (DU) during installation.
```

Comment:
Not sure you need _This procedure tells you how to_. If you want to keep, maybe change to _This section_? It isn't a single procedure. 
Do we need to define what a DU is?

Reply:
Changed to "The procedures in this topic..." / "The procedures do not describe..." / "Rather, they are intended..."